src/main/java/com/iyzipay/Base64FileHelper.java

```diff
@@ -0,0 +1,37 @@
+package com.iyzipay;
+
+import com.iyzipay.exception.ImageFileEncodingException;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+public final class Base64FileHelper {
+
+    private Base64FileHelper() {
+    }
+
+    public static String encode(String filePath) throws ImageFileEncodingException {
```

Comment:
encode metodu başka dosya türleri için de kullanılamaz mı? Image diyerek sadece image dosyasının encode edileceğini varsaymış olmuyor muyuz?

Reply:
```FileBase64Encoder``` adiyla guncelledim. Image'a ait hicbir context yok classta. exception da ```FileEncodingException``` oldu.